tests/filePush/filePush.test.ts

```diff
@@ -0,0 +1,33 @@
+const mockConnect = jest.fn();
+const mockFastPut = jest.fn();
+const mockEnd = jest.fn();
+
+jest.mock('ssh2-sftp-client', () => {
+  return { 
+    __esModule: true,
+    default: jest.fn().mockImplementation(() =>{
+      return { 
+        connect: mockConnect,
+        fastPut: mockFastPut,
+        end: mockEnd,
+      };
+    }),
+  };
+});
+
+import { filePush } from '../../src/filePush/filePush';
+
+describe('test the push to SFTP server', () => {
```

Comment:
I think the describe normally has the function/class/ file name explicitly but there's no standards around this